frontend/package.json

```diff
@@ -65,7 +65,8 @@
     "react-window": "^1.8.8",
     "ts-jest": "^29.1.0",
     "typedoc": "^0.23.26",
-    "uuid": "^9.0.0"
+    "uuid": "^9.0.0",
+    "tui-calendar": "^1.15.3"
```

Comment:
If I remember correctly we removed tui-calendar dependency because we switched to own implementation. @LanderDurie @Joris-Peeters do you still require this dependency?